stable/metricbeat/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+description: A Helm chart to collect Kubernetes logs with metricbeat
+icon: https://www.elastic.co/assets/blt47799dcdcf08438d/logo-elastic-beats-lt.svg
+name: metricbeat
+version: 0.3.0
+appVersion: 6.2.3
```

Comment:
maybe go straight for latest 6.4.0?
If so - change values / Chart / readme to make the value consistent

Reply:
i replaced version number to 6.4.0, thx!